Ecommerce/index.html

```diff
@@ -8,6 +8,13 @@
 
   <body style="margin: 0; padding: 0">
```

Comment:
The img tag is missing the alt attribute, which is required for accessibility purposes.